### PR TITLE
Standardize VS version

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: docs/core/whats-new/dotnet-core-3-0.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ ms.date: 09/22/2019
...
 
-.NET Core 3.0 adds support for C# 8.0. It's highly recommended that you use [Visual Studio 2019 16.3](https://visualstudio.microsoft.com/vs/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019), [Visual Studio for Mac 8.3](/visualstudio/mac/install-preview), or [Visual Studio Code](https://code.visualstudio.com/) with the **C# extension**.
+.NET Core 3.0 adds support for C# 8.0. It's highly recommended that you use [Visual Studio 2019 version 16.3](https://visualstudio.microsoft.com/vs/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019), [Visual Studio for Mac 8.3](/visualstudio/mac/install-preview), or [Visual Studio Code](https://code.visualstudio.com/) with the **C# extension**.
```
Comment (BillWagner) [inline docs/core/whats-new/dotnet-core-3-0.md]: We should probably future proof this statement:

```suggestion
.NET Core 3.0 adds support for C# 8.0. It's highly recommended that you use [Visual Studio 2019 version 16.3](https://visualstudio.microsoft.com/vs/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) or newer, [Visual Studio for Mac 8.3](/visualstudio/mac/install-preview) or newer, or [Visual Studio Code](https://code.visualstudio.com/) with the latest **C# extension**.
```

File: docs/visual-basic/getting-started/whats-new.md [hunk 2]
```diff
@@ -15,45 +15,45 @@ This topic lists key feature names for each version of Visual Basic, with detail
 
 ## Current version
 
-Visual Basic 16.0 / Visual Studio 2019 Version 16.0  
+Visual Basic 16.0 / Visual Studio 2019 version 16.0
```
Comment (BillWagner) [inline docs/visual-basic/getting-started/whats-new.md]: The whitespace that was removed here was significant.  It was used to generate a hard new line in the rendered page.  See [current site](https://docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/whats-new) vs. [this change](https://review.docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/whats-new?branch=pr-en-us-15291)

That applies to all changes in this file.

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall, this is great @gewarren 

I had one suggestion on future-proofing the versions, and one formatting concern. Once those are fixed, this is ready to :shipit:

File: docs/visual-basic/getting-started/whats-new.md (hunk 2 above)
Comment (gewarren) [reply docs/visual-basic/getting-started/whats-new.md]: Oopsies. Would it be okay if I add a hard line break using a \ character?

Comment (BillWagner) [reply docs/visual-basic/getting-started/whats-new.md]: I think so. @mairaw might have a preferred suggestion.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks @gewarren I've approved the latest changes.

**Pull request merged**